infra/src/main/java/com/redhat/service/smartevents/infra/exceptions/BridgeErrorInstance.java

```diff
@@ -0,0 +1,56 @@
+package com.redhat.service.smartevents.infra.exceptions;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@Schema
+public class BridgeErrorInstance extends BridgeError {
+
+    private String uuid;
+
+    protected BridgeErrorInstance() {
+        //(De-)serialisation
+    }
+
+    public BridgeErrorInstance(BridgeError bridgeError) {
+        super(bridgeError.getId(), bridgeError.getCode(), bridgeError.getReason(), bridgeError.getType());
+        this.uuid = UUID.randomUUID().toString();
+    }
+
+    @JsonIgnore
+    public String getUUID() {
+        return uuid;
+    }
```

Comment:
I can't remember by heart so I might be wrong. We are going to include the `uuid` in the serialized object right? If so, could you use `@JsonProperty` on the `uuid`? I don't remember exactly the magic behavior of Jackson in this case, but if you remove `@JsonIgnore` the result should be the same right?

Reply:
Done (and now consistent with our other DTOs).

Reply:
I had to add back the `@JsonIgnore` otherwise I was getting two properties in the Open API definition:
```
uuid:
  type: string
uUID:
  type: string
```